### DISCUSSION TITLE
No code reviews.

### DISCUSSION THREAD

**Discussion by NiceBoy0407** [community/community#51752] [Pull Requests] [answered]
### Select Topic Area

Question

### Body

Hi everyone!
I'm making a pull request and leaving a comment, but the code review history doesn't appear on the dashboard~~~.

please help me.

**Comment by joneslee0918** [comment, marked as answer]
I think you didn't understand the pull request and the code review clearly.
Check out this guide. maybe that will help.
https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request

**Comment by NiceBoy0407** [comment]
thanks for your effors.
